build.gradle

```diff
@@ -411,8 +409,8 @@ def createSymlinks(archivePath, symlinkLocation) {
 def createGatkSymlinks(destinationDir, archivePath, suffix, baseJarName, secondaryBaseJarName) {
     def finalSuffix = (suffix == "") ? "" : ("-" + suffix)
 
-    def symlinkLocation = destinationDir.toString() + "/" + baseJarName + finalSuffix + ".jar"
-    def symlinkLocation2 = destinationDir.toString() + "/" + secondaryBaseJarName + finalSuffix + ".jar"
+    def symlinkLocation = destinationDir.getAsFile().get().toString() + "/" + baseJarName + finalSuffix + ".jar"
```

Comment:
this one is strange but I don't know the intended magic way to do it

Reply:
Yeah -- the destinationDir is a DirectoryProperty which you can turn into a Provider<File> with `getAsFile()` but to resolve that Provider -> File you call `.get()`